app/controllers/equipment_models_controller.rb

```diff
@@ -192,4 +193,21 @@ def fix_content_type(filedata)
   def type_from_file_command(file)
     Paperclip::FileCommandContentTypeDetector.new(file).detect
   end
+
+  def generate_calendar_reservations
+    # we need uniq because it otherwise includes overdue reservations in the
+    # date range twice
```

Comment:
this comment should also probably be included in the categories and items controller implementations